module/platform/cm740/HardwareIO/README.md

```diff
@@ -13,20 +12,14 @@ aborted.
 
 This module reads the current status of the CM740 90 times per second and
 emits it as a `message::platform::RawSensors` object. This includes the CM740 error
-code, LED panel, head and eye LED colour, buttons, voltage, accelerometer,
-gyroscope, left and right force-sensing resistors and each servo.
-
-To change the colour of the NUgus' head or eye LEDs, emit a
-`message::platform::RawSensors::EyeLED` or `message::platform::RawSensors::HeadLED`
-containing the colour you wish to set them to.
```

Comment:
The functionality still exists (the CM740 still has the hardware connections available to connect up these LEDs), not the modules fault that we don't take advantage of it.